src/sentry/integrations/opsgenie/actions/form.py

```diff
@@ -47,24 +56,63 @@ def __init__(self, *args, **kwargs):
         self.fields["team"].widget.choices = self.fields["team"].choices
 
     def _team_is_valid(
-        self, team_id: str | None, org_integration: RpcOrganizationIntegration | None
-    ) -> bool:
-        return True if get_team(team_id, org_integration) is not None else False
+        self,
```

Comment:
would this be more accurately titled `_get_team_status` or sumthin (since it doesn't return a boolean)